src/interactivity/win32/window.cpp

```diff
@@ -314,16 +314,16 @@ void Window::_UpdateSystemMetrics() const
 
             if (useDx)
             {
-                status = NTSTATUS_FROM_HRESULT(pDxEngine->SetHwnd(hWnd));
+                status = NTSTATUS_FROM_WIN32(HRESULT_CODE((pDxEngine->SetHwnd(hWnd))));

```

Comment:
y tho?

Reply:
Michael and I were trying to debug something some time ago. This was a remnant of that.